training-vm/Vagrantfile

```diff
@@ -0,0 +1,34 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby:
+
+Vagrant.configure("2") do |config|
+
+  config.vm.box = "dummy"
+  config.vm.box_url="https://github.com/mitchellh/vagrant-aws/raw/master/dummy.box"
+
+  config.vm.provider :aws do |aws, override|
+    aws.access_key_id = ENV['AWS_ACCESS_KEY_ID']
+    aws.secret_access_key = ENV['AWS_SECRET_ACCESS_KEY']
+    aws.keypair_name = "govuk-training-integration"
+    aws.ami = "ami-a192bad2"
+    aws.tags["Name"]= "training"
+    aws.block_device_mapping = [ 
+      {
+        'DeviceName' => "/dev/sda1",
+        'Ebs.VolumeSize' => 100,
+        'Ebs.DeleteOnTermination' => true
+      },
+    ]
+    aws.iam_instance_profile_name = ENV['AWS_IAM_INSTANCE_PROFILE_NAME']
+    aws.region = ENV['AWS_REGION']
```

Comment:
AMIs are locked to a region so either the AMI should be configurable (or, ideally, the latest imaged fetched) or the region should be locked to `eu-west-1`.

I've opened a support ticket to recreate default subnets on our default VPC so that (should) remove the need to manually set the `subnet_id`.

Otherwise this looks good to me.